mobile/scripts/fetch_release.sh

```diff
@@ -12,19 +11,12 @@ curl -L -o textile-go_v${VER}_ios-framework.tar.gz https://github.com/textileio/
 tar xvfz textile-go_v${VER}_ios-framework.tar.gz
 rm textile-go_v${VER}_ios-framework.tar.gz
 mv Mobile.framework ${IOS_DIR}
-mv protobuf_gen ${IOS_DIR}
+mv protos ${IOS_DIR}
 
 # fetch Android framework
 mkdir -p ${ANDROID_DIR}
 curl -L -o textile-go_v${VER}_android-aar.tar.gz https://github.com/textileio/textile-go/releases/download/v${VER}/textile-go_v${VER}_android-aar.tar.gz
 tar xvfz textile-go_v${VER}_android-aar.tar.gz
 rm textile-go_v${VER}_android-aar.tar.gz
 mv mobile.aar ${ANDROID_DIR}
-mv protobuf_gen ${ANDROID_DIR}
-
-# fetch JS types
```

Comment:
Removed the fetching of the JS types because they are included in the module (built by our fork of protobufjs).